Workflows/GIT/Branches.md

```diff
@@ -0,0 +1,33 @@
+Our Git process is an upgraded version of [Github Flow](https://guides.github.com/introduction/flow/).
+
+![](https://ftp.infinum.co/stjepan_hadjic/git-flow-2.jpg)
+
+Instead of a single `master` branch, we use two branches to record the history of the project. The `master` branch stores code that's currently in production. The `staging` branch serves as an integration branch for features and fixes. It holds data that reflects what's currently on staging.
+
+**The master branch is always deployable.**
```

Comment:
I would change this to `should always be deployable` or `must always be deployable` to emphasize it a bit